nexus-repository-conan/src/main/java/org/sonatype/repository/conan/rest/ConanProxyRepositoriesApiResourceBeta.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2017-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.repository.conan.rest;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+import javax.ws.rs.Path;
+
+import org.sonatype.nexus.repository.rest.api.AbstractRepositoryApiRequestToConfigurationConverter;
+import org.sonatype.nexus.repository.rest.api.AuthorizingRepositoryManager;
+import org.sonatype.nexus.repository.rest.api.RepositoriesApiResourceBeta;
+import org.sonatype.repository.conan.api.ConanProxyRepositoryApiRequest;
+
+import io.swagger.annotations.Api;
+
+/**
+ * @since 1.0.18
+ */
```

Comment:
For the classes with @Deprecated annotation add pls class header with Description:
```
@deprecated the 'beta' prefix is being phased out, prefer starting new APIs with {@link APIConstants#V1_API_PREFIX} instead. 
```